Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentProducer/local.settings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "apiKey": "11111111-2222-3333-4444-555555555555",
+        "startDaysAgo": "-30",
+        "connectStr": "your azure redis cache connection string",
+        "workspace": "your instance name",
+        "CohesityQueueName": "your CohesityQueueName",
+        "DefaultEndpointsProtocol": "your DefaultEndpointsProtocol",
```

Comment:
Where do you use it? Remove it if you don't.

Reply:
are you sure? FYI, this is its value, I am afraid, it is used by function.        
        "DefaultEndpointsProtocol": "http;AccountName=devstoreaccount1",

Reply:
We take the whole connection string, including the protocol as a parameter. For ex, here's how I form it in azuredeploy.json 
`        "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ProducerFunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ProducerFunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",`
As you can see, I don't take protocol from another variable, and we never change it.